src/style.css

```diff
@@ -0,0 +1,106 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+body {
+  --bigIconSize: 19px;
+  --smallIconSize: 11px;
+}
+
+.search {
+  display: flex;
+  margin-bottom: 20px;
+  position: relative;
+}
+
+.search--big {
+  height: 70px;
+  margin-top: 20px;
+}
+
+.search--small {
+  height: 42px;
+  margin-top: 20px;
+}
+
+.search input {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  background: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  font-family: Avenir, sans-serif;
+}
+
+.search--big::before {
+  content: "";
+  background-image: url(images/Search.svg);
+  position: absolute;
+  z-index: 999;
+  background-size: 19px;
+  top: 25px;
+  left: 26px;
+  height: 19px;
+  width: 19px;
+
+}
+
+.search--small::before {
+  content: "";
```

Comment:
Selectors `.search--big::before` and `.search--small::before` have the same rules.
This breaks the rule [Don't repeat yourself](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).
For a solution check [selector lists](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Selectors#selector_lists).

Reply:
For selectors `.search--big input` and `.search--small input` same thing.